docs/stellar-disbursement-platform/getting-started.mdx

```diff
@@ -235,26 +235,36 @@ In `docker-compose-sdp-anchor.yml`, update the following:
 <CodeExample>
 
 ```yaml
-# the public key of the account used for authentication
-# it should be the same public key for both attributes
-SEP10_ACCOUNT:
+# the public key of the Stellar account used for SEP-10 authentication
 SEP10_SIGNING_PUBLIC_KEY:
-# the private key of the account used for authentication
-# it should be the same secret key for both attributes
+#
+# the private key of the Stellar account used for SEP-10 authentication
+# it should be the same secret key for both attributes, in SDP and AP
 SEP10_SIGNING_PRIVATE_KEY:
 SECRET_SEP10_SIGNING_SEED:
-# the encryption key (any secret string) for the authentication token shared with wallets
-# it should be the same key for both attributes
-SEP24_JWT_SECRET:
+#
+# The secret used to sign the resulting SEP-10 JWT token. This should be a strong random string for encryption.
 SECRET_SEP10_JWT_SECRET:
-# the encryption key for the authentication token used for internal services
-# These could be different or the same value
-ANCHOR_PLATFORM_OUTGOING_JWT_SECRET:
+#
+# A shared encryption key used to sign JWT tokens in the SEP24 communications from AP->SDP.
```

Comment:
Again, defining AP and SDP would be beneficial for clarity.

Reply:
Addressed in https://github.com/stellar/stellar-docs/pull/239/commits/26f2f7ff0bcc7ca3d1f8d59927095e26e9e4d379.